src/main/java/eu/xenit/gradle/docker/DockerBuildBehavior.java

```diff
@@ -102,6 +105,12 @@ public void execute(Project project) {
 
         Task task = project.getTasks().getAt("composeUp");
         task.dependsOn(buildDockerImage);
+
+        DockerRemoveImage cleanDockerImage = project.getTasks().create("cleanDockerImage", DockerRemoveImage.class);
+        cleanDockerImage.setGroup(GRADLE_DOCKER_TASK_GROUP);
+        cleanDockerImage.targetImageId(buildDockerImage.getImageId());
+        cleanDockerImage.getForce().set(true);
+        cleanDockerImage.dependsOn(buildDockerImage);
```

Comment:
`dependsOn` is not necessary, I think. `mustRunAfter` might be more appropriate here.

Because it does not make much sense to run `buildDockerImage` when you are calling `cleanDockerImage` by itself. `mustRunAfter` does not add an explicit dependency, it only affects ordering in the task graph when both tasks are in the graph.